Slim/Menu/TrackInfo.pm

```diff
@@ -396,25 +396,10 @@ sub infoContributors {
 	if ( $track->isa('Slim::Schema::Track') ) {
 		my @roles = Slim::Schema::Contributor->contributorRoles;
 
-		# Loop through each pref to see if the user wants to link to that contributor role.
-		my %linkRoles = map {$_ => $prefs->get(lc($_) . 'InArtists')} @roles;
-		$linkRoles{'ARTIST'} = 1;
-		$linkRoles{'TRACKARTIST'} = 1;
-		$linkRoles{'ALBUMARTIST'} = 1;
-
 		# Loop through the contributor types and append
 		for my $role ( @roles ) {
 			for my $contributor ( $track->contributorsOfType($role) ) {
-				if ($linkRoles{$role}) {
-					$_addContributorItem->($items, $contributor, $role);
-				} else {
-					my $item = {
-						type    => 'text',
-						name    => $contributor->name,
-						label   => uc $role,
-					};
-					push @{$items}, $item;
-				}
```

Comment:
Are you saying that previously we would actually _prohibit_ access to those roles from the menus?..

Reply:
They weren't hyperlinked (just text) in the AlbumInfo and TrackInfo header html unless they were "standard" roles.

Reply:
See the images I posted above: the additional roles are not bolded hyperlinks, just plain text.